cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -428,7 +444,19 @@ BEGIN
                         SUM(w.service_connections_count)       AS service_connections_count,
                         SUM(w.population_served)               AS population_served
                  FROM water_systems w
-                 WHERE ST_Transform(w.bbox, 3857) && ST_TileEnvelope(z, x, y)
+                 WHERE
+                    ST_Transform(w.bbox, 3857) && ST_TileEnvelope(z, x, y) AND
```

Comment:
Same here re: SRID.

Reply:
I think this one is legitimate.  `ST_TileEnvelope` always returns geometry in SRID 3857 (according to https://postgis.net/docs/ST_TileEnvelope.html), so IMO we should project to the same SRID for comparison. 

Does that seem fair?

Reply:
SGTM. Can you add a comment with that info?

Reply:
Done. We actually legitimately use both 3857 and 4326 in several places, so I've added a block comment at the top explaining why and when we should use each one.